ui/src/utils/private/css-class.js

```diff
@@ -0,0 +1,16 @@
+export function getCssClassesAsObject (cssClasses) {
+  if (typeof cssClasses === 'string') {
+    return cssClasses
+      .split(' ')
+      .reduce((acc, curr) => ({ ...acc, [ curr ]: true }), {})
```

Comment:
I updated this to add each class as a single property, to ease further manipulation of the classes after someone used this helper